locale/en_US/locale.xml

```diff
@@ -24,4 +24,8 @@
 	<message key="plugins.themes.classic.register.privacyConsent">Yes, I agree to have my data collected and stored according to the journal's Policy Statement:</message>
 	<message key="plugins.themes.classic.register.privacyConsentContext"><![CDATA[The <a href="{$privacyUrl}" target="_blank">Policy Statement</a> of {$privacyContextName}.]]></message>
 
+	<message key="plugins.themes.classic.options.journalSummary.description">If enabled, the journal summary will be shown on the journal’s landing page.</message>
+	<message key="plugins.themes.classic.options.journalSummary.disable">Disable (default)</message>
+	<message key="plugins.themes.classic.options.journalSummary.enable">Enable</message>
```

Comment:
With radio buttons, it's best to make each option descriptive. So remove the description and change the options to:

- [ ] Do not show the journal summary on the journal's homepage.
- [ ] Show the journal summary on the journal's homepage.

Also, you don't need to add (default) because if it is a default it will be automatically selected when the form loads.